intelligent title (#56)
avo-hq/avodocs: docs/3.0/resources.md

:::option `self.title`

> The resource possesses an intelligent method for retrieving the title. It attempts to invoke a series of record methods in the following order: `name`, `title`, `label`, and `id`.

Reviewer: ```suggestion
Each Avo resource will try to figure out what the title of a record is. It will try the following attributes in order `name`, `title`, `label`, and fallback to the `id`.
```

  end
end
```

> ### Lambda title

Reviewer: ```suggestion
### `title` as a block
```

> If you prefer not to utilize any record methods and instead compute the resource's title directly within the resource itself, you can accomplish this by assigning a lambda function to the `title` class attribute. You'll have access to `resource` and `record`.

Reviewer: ```suggestion
If you prefer not to use any record methods and instead compute the resource's title directly within the resource itself, you can accomplish this by assigning a lambda function to the `title` class attribute. You'll have access to `resource` and `record`.
```
